TableView/Models/ConfigureCellModel.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ConfigureModel.swift
+//  TableView
+//
+//  Created by Oleg Yankiwskyi on 4/16/18.
+//  Copyright © 2018 Oleg Yankiwskyi. All rights reserved.
+//
+
+import Foundation
+
+class ConfigureCellModel: ConfigureCellProtocol {
+    func configure(value: String, model: DataStructureTableCell) {
+        model.label.text = value
```

Comment:
cell isn't model itself
Cell is always view
I'm not sure if we need separate class for this particular case